shared/profile/edit-avatar/container.js

```diff
@@ -36,17 +44,33 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     dispatch(ProfileGen.createUploadAvatar({crop, filename})),
 })
 
-const mergeProps = (stateProps, dispatchProps, ownProps) => ({
-  createdTeam: stateProps.createdTeam,
-  image: stateProps.image,
-  onClose: dispatchProps.onClose,
-  onSave: (filename: string, crop?: RPCTypes.ImageCropRect) =>
-    stateProps.teamname
-      ? dispatchProps.onSaveTeamAvatar(filename, stateProps.teamname, stateProps.sendChatNotification, crop)
-      : dispatchProps.onSaveUserAvatar(filename, crop),
-  sendChatNotification: stateProps.sendChatNotification,
-  teamname: stateProps.teamname,
-})
+const networkErrorCodes = [
+  RPCTypes.constantsStatusCode.scgenericapierror,
+  RPCTypes.constantsStatusCode.scapinetworkerror,
+  RPCTypes.constantsStatusCode.sctimeout,
+]
+const mergeProps = (stateProps, dispatchProps, ownProps) => {
+  let error = ''
+  if (stateProps.error) {
+    error = networkErrorCodes.includes(stateProps.error.code)
```

Comment:
Basic check for if it was a network error. Assumes it is an unsupported file type if not.